web/src/lib/components/shared-components/settings/setting-switch.svelte

```diff
@@ -30,9 +33,8 @@
       {/if}
     </div>
 
-    <p class="text-sm dark:text-immich-dark-fg">{subtitle}</p>
-    <slot />
```

Comment:
Not safe to remove because the slot is used here:

https://github.com/immich-app/immich/blob/4ed68cf67385efac66d5c5035cd79f849a36959c/web/src/lib/components/user-settings-page/app-settings.svelte#L77-L84